index.html

```diff
@@ -1024,16 +1024,16 @@ <h2>Extract response headers</h2>
       Given a <a>network request</a> (<var>request</var>) and its corresponding
       <a>response</a> (<var>response</var>), this algorithm generates a report
       about <var>request</var> if instructed to by any matching <a>NEL
-        policy</a>, and queues it for delivery.
+        policy</a>.

```

Comment:
Please mention here that the algorithm can return null instead.

Reply:
done